package.json

```diff
@@ -29,12 +29,14 @@
   },
   "homepage": "https://github.com/rgbkrk/pipboy#readme",
   "devDependencies": {
+    "babel-preset-es2015": "^6.1.18",
+    "electron-compile": "^1.0.1",
```

Comment:
This will probably need to be a direct dependency as it's used in main, for when we package for wider distribution.